GoogleChromeLabs/ps-analysis-tool
# Enhancement: CLI decoupling - Save analysis results in a directory

## Description

This enhances the CLI tool's interface by adding a new argument `-d <path-to-dir>` or `--out-dir <path-to-dir>`. If this argument is provided the CLI tool will not open the analysis result in the dashboard. The results rather will be saved in the provided directory.

For single site input, the output directory will have the following files.
- `affected-cookies.csv`
- `cookies.csv`
- `report.csv`
- `technologies.csv`
- `out.json`

For multiple site input cases ( a sitemap or a CSV file ), the output directory will have directories for each URL with the files mentioned above.

## Testing Instructions

- Follow instructions on using CLI in the README.md.
- Create a directory for storing analysis result.
- Add a new argument for example - `npm run cli -- -u https://example.com -np -d ./foo`.
- The directory `foo` should have the files mentioned in the description of the PR.




## Checklist






Partially Addresses #203
